elastic/kibana
# [Infra UI] Hosts view: Enable Anomalies, Osquery tabs and open as page link inside asset details flyout

Closes  #160653

## Summary

This PR enabled Anomalies, Osquery tabs, removes Uptime link and adds open as page link.

## Testing
- Before going to host view go to inventory click on Anomaly detection and and set up ML jobs for hosts (it will be enable instead of recreate in case it's not enabled:
<img width="200" alt="image" src="https://github.com/elastic/kibana/assets/14139027/29eb5112-7292-474a-86f3-7e9a765ac307"> )
- Go to the hosts view and open the flyout

  - the Anomalies Tab should be visible (find a host with detected anomaly and open it):
  <img width="1728" alt="Screenshot 2023-07-06 at 14 49 08" src="https://github.com/elastic/kibana/assets/14139027/e2d72725-ee3d-4f20-a7f3-c8bed104ce8b">
  
   - Osquery tab should be visible:
    <img width="1728" alt="Screenshot 2023-07-06 at 14 41 30" src="https://github.com/elastic/kibana/assets/14139027/114d6e94-0792-4af6-8755-1c0d0d96176d">
   
   - Click on open as page - should navigate to node details page
